### PR TITLE
Programmes 6675 episodes that have no parent show number of programmes for each topic displayed

### PR DESCRIPTION
**Pull request by cisco87**
https://jira.dev.bbc.co.uk/browse/PROGRAMMES-6675

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

File: src/Controller/FindByPid/EpisodeController.php [hunk 1]
```diff
@@ -176,6 +175,7 @@ public function __invoke(
             'episodeMapPresenter' => $episodeMapPresenter,
             'segmentsListPresenter' => $segmentsListPresenter,
             'podcastedBy' => ($podcast !== null) ? $episode->getTleo() : null,
+            'isPartOfASeries' => !is_null($episode->getPosition()),
```
Comment (votemike) [inline src/Controller/FindByPid/EpisodeController.php]: Is this used?

File: src/Ds2013/Presenters/Section/RelatedTopics/RelatedTopicsPresenter.php [hunk 2]
```diff
@@ -33,4 +40,9 @@ public function getRelatedTopics(): array
     {
         return $this->relatedTopics;
     }
+
+    public function showCount(): bool
+    {
+        return !($this->context instanceof Episode && $this->context->isTleo());
```
Comment (votemike) [inline src/Ds2013/Presenters/Section/RelatedTopics/RelatedTopicsPresenter.php]: This seems a bit hard to read. So we should show the count if it's a Clip and not a TLEO?

File: src/Controller/FindByPid/EpisodeController.php (hunk 1 above)
Comment (cisco87) [reply src/Controller/FindByPid/EpisodeController.php]: Nope it was part of another experiment

File: src/Ds2013/Presenters/Section/RelatedTopics/RelatedTopicsPresenter.php (hunk 2 above)
Comment (cisco87) [reply src/Ds2013/Presenters/Section/RelatedTopics/RelatedTopicsPresenter.php]: This logic is tested and has a behaviour compatible with V2, I will make it easier to read

**Comment by tatitati** [comment]
long branch name! XD

**Comment by cisco87** [comment]
> long branch name! XD

alias branch='function _branch() { echo $1 | sed -e "s/[^a-zA-Z0-9]/\-/g"; }; _branch'
–(~)–($ branch 'This is a test with a long jira ticket title'  
This-is-a-test-with-a-long-jira-ticket-title

:D

**Pull request merged**